addons/survey/tests/test_survey_invite.py

```diff
@@ -258,3 +258,31 @@ def test_survey_invite_token_by_email_nosignup(self):
         answers = self.env['survey.user_input'].search([('survey_id', '=', self.survey.id)])
         self.assertEqual(len(answers), 1)
         self.assertEqual(answers.partner_id.display_name, first_partner.display_name)
+
+    @users('survey_manager')
```

Comment:
Could you please use `survey_user` instead? This should be working for them too, and the manager inherits (yes even though it isn't already like this in the file ;-))

Reply:
The `survey_user` weirdly has an ir.rule that takes away their create perms on the `survey.user_input` model with (1, =, 1) domain constraint (even though they are granted read,write,link,unlink in model.access.csv) so they are unable to call `action_invite()` without raising an AccessError

Reply:
The rule is that they can create user inputs on survey they created (a bit weird by the way). So probably you should check that survey_user create a survey then invites people on it ?